python/examples/colors.py

```diff
@@ -22,6 +22,8 @@
 anisotropy = []
 
 for frame in frames:
+    # center in the box
+    frame.positions += frame.cell.diagonal() * 0.5
```

Comment:
What's the purpose of this here?

Reply:
For reasons I didn't investigate, if the molecule is at the origin it'll be shown folded into the box (i.e. split into pieces). this was to avoid it without figuring out this bug in this PR, because I'm scared Rose, very scared, by what you'd do to me otherwise.